### PR TITLE
Refine styles for query chips

### PR DESCRIPTION
**Pull request by quinnkeast**
## Test plan

Review styles and interaction behavior visually.

## App preview:

- [Web](https://sg-web-qk-query-examples-refinements.onrender.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

File: client/web/src/search/home/QueryExamplesHomepage.module.scss [hunk 1]
```diff
@@ -46,6 +53,16 @@
     }
 }
 
+.query-examples-items {
```
Comment (quinnkeast) [inline client/web/src/search/home/QueryExamplesHomepage.module.scss:56]: This resolves an extra margin-bottom affecting the chip in the "Tip."

File: client/web/src/search/home/QueryExamplesHomepage.tsx [hunk 2]
```diff
@@ -56,8 +56,8 @@ const queryExampleSectionsColumns = [
             title: 'Get advanced',
             queryExamples: [{ id: 'contains-commit-after', query: 'repo:contains.commit.after(yesterday)' }],
             footer: (
-                <small className="d-block mt-2">
-                    <Link target="blank" to="/help/code_search/reference/queries">
+                <small className="d-block mt-3">
+                    <Link target="blank" to="https://docs.sourcegraph.com/code_search/reference/queries">
```
Comment (quinnkeast) [inline client/web/src/search/home/QueryExamplesHomepage.tsx:60]:
```suggestion
                    <Link target="blank" to="/help/code_search/reference/queries">
```

**Review by novoselrok — APPROVED** [review]

File: client/web/src/search/home/QueryExamplesHomepage.module.scss [hunk 3]
```diff
@@ -46,6 +53,16 @@
...
+    & > * {
```
Comment (novoselrok) [inline client/web/src/search/home/QueryExamplesHomepage.module.scss:57]: You can remove the `&` here.

Comment (quinnkeast) [inline client/web/src/search/home/QueryExamplesHomepage.module.scss:57]:
```suggestion
    > * {
```

**Pull request merged**